QRCoder/Builders/Renderers/ITextRenderer.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.IO;
+
+namespace QRCoder.Builders.Renderers
+{
+    public interface ITextRenderer
+    {
+        string ToString();
+    }
```

Comment:
Concept is that all string-based renderers (Ascii, Svg) implement this interface.